account/routes.js

```diff
@@ -16,11 +17,16 @@ router.get('/sign-in', async (req, res, next) => {
             referrer: req.get('Referrer') || `${req.protocol}://${req.get('host')}/apply/`
         };
         const encodedState = Buffer.from(JSON.stringify(stateObject)).toString('base64');
-
+        if (req.session.nonce) {
```

Comment:
If we are always setting the session property, there is no need to unset it just before.